src/cases/containers/allocate-role-container/allocate-role-container.component.html

```diff
@@ -0,0 +1 @@
+<exui-allocate-role [roleAllocation]="roleAllocationType" [includeOther]="includeOther"></exui-allocate-role>
```

Comment:
would this best to move to src\work-allocation-2\containers, if this is only used in work allocation 2.